src/Goteo/Util/ModelNormalizer/Transformer/ImpactDataTransformer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/*
+ * This file is part of the Goteo Package.
+ *
+ * (c) Platoniq y Fundación Goteo <fundacion@goteo.org>
+ *
+ * For the full copyright and license information, please view the README.md
+ * and LICENSE files that was distributed with this source code.
+ */
+namespace Goteo\Util\ModelNormalizer\Transformer;
+
+use Goteo\Core\Model;
+use Goteo\Library\Text;
+
+/**
+ * Transform a Model
+ */
```

Comment:
In my opinion, the class name is quite self explanatory and this comment is not adding value. If you agree, I'd vote to simply remove the comment.